docs/next/components/mdx/RenderedDAG.tsx

```diff
@@ -0,0 +1,380 @@
+import animate from 'amator';
+import React from 'react';
+
+const MAX_WIDTH = 500;
+const MAX_HOVER_WIDTH = (MAX_WIDTH / 40.0) * 40.5;
+
+// Tweaked version of the website component
+// See https://github.com/dagster-io/dagster-website/blob/master/components/RenderedDAG.tsx
+
+export const RenderedDAG: React.FC<{
+  svgSrc: string;
+  isAssetGraph?: boolean;
+  mobileImgSrc: string;
+}> = ({svgSrc}) => {
+  const [focus, setFocus] = React.useState(false);
+  const [hover, setHover] = React.useState(false);
+  const [content, setContent] = React.useState('');
+
+  React.useEffect(() => {
+    const load = async () => {
+      const result = await fetch(svgSrc);
+      const text = await result.text();
+      setContent(text);
+    };
+    load();
+  }, [svgSrc]);
+
+  return (
+    <div
+      className="text-black hidden lg:block"
+      style={{
+        minHeight: 400,
+        width: '40.5%',
+        maxWidth: MAX_HOVER_WIDTH,
+        boxSizing: 'border-box',
+        flexShrink: 0,
+      }}
+    >
+      <div
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+        style={{
+          lineHeight: 1.1,
+          position: 'absolute',
+          bottom: 0,
+          top: 0,
+          right: 0,
+          paddingLeft: focus ? 0 : hover ? 5 : 15,
+          width: focus ? '100%' : '40.5%',
+          maxWidth: focus ? '100%' : MAX_HOVER_WIDTH,
+          left: 'unset',
+          cursor: focus ? 'grab' : 'zoom-in',
+          zIndex: focus || hover ? 2 : 1,
+          transition:
+            'width 300ms ease-in-out, max-width 300ms ease-in-out, padding-left 300ms ease-in-out, left 300ms ease-in-out',
+        }}
+      >
+        {/* {isAssetGraph && <MaterializeButton />} */}
```

Comment:
I wonder if we could go ahead and remove the `isAssetGraph` prop? I could see wiring it up to whether or not the image URL contains `-assets` or something like that, but it looks like it's a no-op now that this is commented out (which makes sense, all the buttons could be confusing in the docs)